src/main/java/com/xiaomi/infra/pegasus/client/ClientOptions.java

```diff
@@ -63,6 +65,8 @@
   public static final Duration DEFAULT_META_QUERY_TIMEOUT = Duration.ofMillis(5000);
   public static final String DEFAULT_AUTH_PROTOCOL = "";
   public static final long DEFAULT_SESSION_RESET_SECS_WINDOW = 30;
+  public static final int DEFAULT_MAX_FETCH_COUNT = 100;
```

Comment:
Actually, the option is `api-level`, shouldn't define into `client-level` config. or you  rename it into

```suggestion
  public static final int DEFAULT_MULTI_GET_MAX_FETCH_COUNT = 100;
```

Reply:
ok

Reply:
Don't suggest, user will misunderstand if every `get` api will all limit it. `clientOption` is a `global` option. I actually suggest you can delete it from here.